staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/buildmanagerinfo.go

```diff
@@ -51,12 +51,12 @@ func (f *buildManagerInfoManager) Update(liveObj, newObj runtime.Object, managed
 }
 
 // Apply implements Manager.
-func (f *buildManagerInfoManager) Apply(liveObj runtime.Object, patch []byte, managed Managed, manager string, force bool) (runtime.Object, Managed, error) {
+func (f *buildManagerInfoManager) Apply(liveObj, newObj runtime.Object, managed Managed, manager string, force bool) (runtime.Object, Managed, error) {
```

Comment:
Throughout this PR, not a big fan of the name "newObj", since it's what we use in "Update" for the complete new object. That's fine for the result of the merge (what we're going to update with), but I don't think it conveys the exact meaning here. I'd be fine with "appliedObj", "partialObj" or anything like that. Clearly `appliedObj` I think is best.

Reply:
i changed it to `appliedObj`

Reply:
thanks